sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, base_url: str, credential: TokenCredential, **kwargs: Any) ->
         if pipeline is None:
             aad_mode = not isinstance(credential, AppConfigConnectionStringCredential)
             pipeline = self._create_appconfig_pipeline(
-                credential=credential, aad_mode=aad_mode, base_url=base_url, **kwargs
+                credential=credential, credential_scopes=self._credential_scopes, aad_mode=aad_mode, **kwargs
```

Comment:
Why we don't need base_url any more?

Reply:
Agree @xiangyan99, improving typing shouldn't change the code, unless it was a bug, and in that case we should create an issue number explaining what we discovered, so we can explain it in the changelog.